docs/private-networks/how-to/configure/consensus/ibft.md

```diff
@@ -250,29 +250,32 @@ Non-validator nodes don't affect performance and don't count towards the maximum
 
 ## Transitions
 
-The `transitions` genesis configuration item allows you to specify a future block number at which to change IBFT 2.0 network configuration in an existing network. For example, you can update the [block time](#configure-block-time-on-an-existing-network-deployment), [block reward](#configure-block-rewards-on-an-existing-network-deployment), or [mining beneficiary](#configure-the-mining-beneficiary-on-an-existing-network-deployment).
+The `transitions` genesis configuration item allows you to specify a future block number at which to
+change the IBFT 2.0 network configuration in an existing network.
+For example, you can update the [block time](#configure-block-time-on-an-existing-network),
+[block reward](#configure-block-rewards-on-an-existing-network), or
+[mining beneficiary](#configure-the-mining-beneficiary-on-an-existing-network).
 
 :::caution
-
-Do not specify a transition block in the past. Specifying a transition block in the past could result in unexpected behavior, such as causing the network to fork.
-
+Do not specify a transition block in the past.
+Specifying a transition block in the past can result in unexpected behavior, such as causing the
+network to fork.
 :::
 
-### Configure block time on an existing network deployment
+### Configure block time on an existing network
 
 To update an existing network with a new `blockperiodseconds`:
 
-1.  Stop all nodes in the network.
-1.  In the [genesis file](#genesis-file), add the `transitions` configuration item where:
+1. Stop all nodes in the network.
+2. In the [genesis file](#genesis-file), add the `transitions` configuration item where:
```

Comment:
oh! Do we not use the Markdown numbering for Besu docs?

Reply:
We don't have a strict rule on auto vs manual numbering, but the auto numbering was causing format issues in my IDE, so switched to manual numbering to be safe (especially with the heavy use of tabs in these steps).